components/Cards.js

```diff
@@ -20,3 +20,56 @@
 // Add a listener for click events so that when a user clicks on a card, the headline of the article is logged to the console.
 //
 // Use your function to create a card for each of the articles, and append each card to the DOM.
+
+import axios from 'axios'
+    let topics = document.querySelector('div.cards-container')
+
+function makeCard (object) {
+    const cardDiv = document.createElement('div')
+    const cardHeadline = document.createElement('div')
+    const cardAuthor = document.createElement('div')
+    const imgContainer = document.createElement('div')
+    const img = document.createElement('img')
+    const span = document.createElement('span')
+
+
+    cardDiv.classList.add('card')
+    cardHeadline.classList.add('headline')
+    cardAuthor.classList.add('author')
+    imgContainer.classList.add('img-container')
+
+    cardHeadline.innerText = object.headline;
+    img.src = object.authorPhoto;
+    span.textContent = "By " + object.authorName;
+
+
+
+
+    cardDiv.append(cardHeadline)
+    cardDiv.append(imgContainer)
+    cardAuthor.append(img)
+    cardAuthor.append(span)
+    cardDiv.append(cardAuthor)
+
+    topics.append(cardDiv)
+}
+
+
+axios.get('https://lambda-times-api.herokuapp.com/articles')
+.then(function (response) {
+    response.data.articles.bootstrap.forEach(element => {
```

Comment:
looks like you're not displaying cards for all of the articles. you can do that by referencing the other articles by name, e.g. `response.data.articles.javascript` or by using Object.entries
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/entries

The latter isn't something we've gone over in class or that you're expected to know